Cargo.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Polygon Avail Team"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kate-recovery = {git = "https://github.com/maticnetwork/avail.git"}
+kate-recovery = {git = "https://github.com/maticnetwork/avail.git", branch="feature/MAT-3374-app-client-crate"}
```

Comment:
Can you post a link to the branch? Couldn't find it.

Reply:
This branch has just been merged to main, I will revert this change as it is no longer needed.